android/app/src/main/java/poke/rogue/helper/presentation/dex/detail/PokemonDetailActivity.kt

```diff
@@ -148,35 +219,40 @@ class PokemonDetailActivity :
                 progressIndicatorPokemonDetail,
             )
 
-            collapsingToolbarLayoutPokemonDetail?.title =
+            tvPokemonDetailPokemonName.text =
                 stringOf(
                     R.string.pokemon_list_poke_name_format,
                     pokemonDetail.pokemon.name,
                     pokemonDetail.pokemon.dexNumber,
                 )
 
-            tvPokemonDetailPokemonName?.text =
-                stringOf(
-                    R.string.pokemon_list_poke_name_format,
-                    pokemonDetail.pokemon.name,
-                    pokemonDetail.pokemon.dexNumber,
-                )
+            if (fullNameChipSpecs.isEmpty()) {
+                fullNameChipSpecs = initialTypeChipSpecs(pokemonDetail)
+            }
+
+            chipGroupPokemonDetailTypes.submitList(fullNameChipSpecs)
```

Comment:
```suggestion
            pokemonDetail.pokemon.types.also {
                types = it
                updateChips()
            }
```

제안 1). uiState 를 collect 하는 시점에서, types 를 멤버 변수에 저장하는건 어떠세요? 그럼, configuration Change도 해결될 듯 합니다.  

제안 2). 기존에 만들어두었던, `updateChipWithLabels()` 를 사용하면 좋을 것 같아요!